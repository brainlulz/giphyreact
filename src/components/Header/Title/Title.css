.Title {
    color: #FFBD39;
    grid-column: 1/1;
    grid-row: 1/1;
    align-self: center;
    justify-self: center;
    font-family: 'Gears of Peace';
    font-size: 5rem;
}

.link {
    color: inherit;
    text-decoration: none;
}