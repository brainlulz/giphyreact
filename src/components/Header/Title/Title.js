import React from "react";
import { NavLink } from 'react-router-dom';
import "./Title.css";

const Title = () => (
    <h1 className="Title">
        <NavLink className="link" to="/">
            Gif or Gif
        </NavLink>
    </h1>
);

export default Title;
