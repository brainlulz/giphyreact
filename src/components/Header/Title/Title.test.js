import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title';

describe('Title', () => {
  it('renders the Title component', () => {
    const wrapper = shallow(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
});
