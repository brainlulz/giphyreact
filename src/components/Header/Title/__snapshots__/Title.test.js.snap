// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders the Title component 1`] = `
<h1
  className="Title"
>
  <Link
    className="link"
    replace={false}
    to="/"
  >
    Gif or Gif
  </Link>
</h1>
`;
